src/Io/PsrMiddlewareAdapter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace FrameworkX\Io;
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface as PsrMiddlewareInterface;
+use React\Promise\PromiseInterface;
+use function React\Async\async;
+
+/**
+ * @internal
+ */
+class PsrMiddlewareAdapter
```

Comment:
This should also be attributed to https://github.com/friends-of-reactphp/http-middleware-psr15-adapter

Reply:
Same as https://github.com/clue/framework-x/pull/253#discussion_r1570850046